lua/nvim-treesitter/ts_utils.lua

```diff
@@ -439,6 +439,14 @@ function M.goto_node(node, goto_end, avoid_set_jump)
   else
     position = { range[3], range[4] }
   end
+
+  -- Enter visual mode if we are in operator pending mode
+  -- If we don't do this, it will miss the last character.
+  local mode = vim.api.nvim_get_mode()
+  if mode.mode == "no" then
+    vim.cmd "normal! v"
+  end
+
```

Comment:
Can you add a test for this please? Just for this function for a case where it was failing before.

Reply:
The test is in the textobjects repo, the one that is failing now!